app/middlewares/generics.js

```diff
@@ -23,3 +24,20 @@ exports.verifyToken = (req, res, next) => {
 
   return next(errors.noTokenProvided);
 };
+
+exports.verifyAdministratorToken = (req, res, next) => {
+  const token = req.body.token || req.query.token || req.headers[config.common.session.header_name];
+
+  if (token) {
+    return jwt.verify(token, config.common.session.secret, (err, decoded) => {
+      if (err) return next(errors.defaultError(err.message));
+
+      if (decoded.permission !== enums.PERMISSION.ADMINISTRATOR)
```

Comment:
if you need to go down a line add the {} , this will be enforced in the new linter rules but for now please change it